<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewPasswordw</class>
 <widget class="QDialog" name="NewPasswordw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 127);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{
color:rgb(255, 255, 255);
	background-color:rgb(255, 170, 127);
}</string>
   </property>
   <property name="title">
    <string>Updating password</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_newpassword">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton{
color:white;
background-color: rgb(170, 0, 127);
border-width:0px;
border-radius:4px;
}
#pushButton:hover{
background-color: rgb(170, 7, 56);
border-width:0px;
border-radius:2px;
}</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
